src/app/gps/qgsappgpsconnection.cpp

```diff
@@ -0,0 +1,177 @@
+/***************************************************************************
+    qgsappgpsconnection.cpp
```

Comment:
We should try to move away from using "GPS" in favor of "GNSS" in our class names / functions. A worthy QGIS 4.0 goal.

Reply:
I've added that to the 4.0 api issues tracker